### PR TITLE
style(frontend): ManageTokensButton is shown only when signed in [GIX-3015]

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

The ManageTokensButton will be shown only when signed in, since it is not much useful visually when signed out.

### Before

Screenshot 2024-09-26 at 18 26 18

### After

Screenshot 2024-09-26 at 18 24 29
Screenshot 2024-09-26 at 18 24 46

### CODE REVIEW

File: src/frontend/src/lib/components/tokens/Tokens.svelte [hunk 1]
```diff
@@ -17,9 +17,9 @@
 
 	{#if $authSignedIn}
 		<TokensSignedIn />
+
+		<ManageTokensButton />
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/tokens/Tokens.svelte:21]: Does it needs a `fade` effect?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/tokens/Tokens.svelte]: Good one! Added `fade` transition (same as token list)


https://github.com/user-attachments/assets/cd8d3fe4-3ab5-4dc1-9642-7e48f1812ba4

File: src/frontend/src/lib/components/tokens/Tokens.svelte [hunk 2]
```diff
@@ -17,9 +18,11 @@
 
 	{#if $authSignedIn}
 		<TokensSignedIn />
+
+		<div transition:fade>
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/tokens/Tokens.svelte:22]: nitpick: `in:` is probably enough

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**